### DISCUSSION TITLE
where can i setting autolink ?

### DISCUSSION THREAD

**Discussion by zhoudaxia2016** [community/community#31912] [Enterprise] [answered]
I can not find the setting in my repo's setting page 
![image](https://user-images.githubusercontent.com/24446852/188304911-a8533c22-0489-4643-aafc-cc8de7f57c36.png)

**Comment by jfullstackdev** [comment]
what is that you are looking at the Settings of the repo ?

**Reply by zhoudaxia2016** [reply]
> what is that you are looking at the Settings of the repo ?

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/configuring-autolinks-to-reference-external-resources

**Reply by jfullstackdev** [reply]
I'm not using it but what I can think of is that, it will be available once a certain configuration is successful,

**Comment by DenverCoder1** [comment, marked as answer]
Autolinks are available in repositories with GitHub Pro, GitHub Team, GitHub Enterprise Cloud, and GitHub Enterprise Server.

[docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/configuring-autolinks-to-reference-external-resources](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/configuring-autolinks-to-reference-external-resources)

You are likely not seeing the option at the bottom of Integrations because the repository is under a Free account.

**Reply by zhoudaxia2016** [reply]
oh so disappointing. thanks to answer me.
